tests/models/llava_onevision/test_modeling_llava_onevision.py

```diff
@@ -320,20 +320,19 @@ def test_generate_compile_model_forward(self):
 
 @require_torch
 class LlavaOnevisionForConditionalGenerationIntegrationTest(unittest.TestCase):
-    def setUp(self):
-        self.processor = AutoProcessor.from_pretrained(
-            "llava-hf/llava-onevision-qwen2-0.5b-ov-hf", padding_side="left"
-        )
+    @classmethod
+    def setUpClass(cls):
+        cls.processor = AutoProcessor.from_pretrained("llava-hf/llava-onevision-qwen2-0.5b-ov-hf", padding_side="left")
         image_file = hf_hub_download(
             repo_id="raushan-testing-hf/images_test", filename="llava_v1_5_radar.jpg", repo_type="dataset"
         )
         video_file = hf_hub_download(
             repo_id="raushan-testing-hf/videos-test", filename="video_demo.npy", repo_type="dataset"
         )
-        self.image = Image.open(image_file)
-        self.video = np.load(video_file)
-        self.prompt_image = "user\n<image>\nWhat do you see in this image?<|im_end|>\n<|im_start|>assistant\n"
-        self.prompt_video = "user\n<video>\nWhat do you see in this video?<|im_end|>\n<|im_start|>assistant\n"
+        cls.image = Image.open(image_file)
+        cls.video = np.load(video_file)
+        cls.prompt_image = "user\n<image>\nWhat do you see in this image?<|im_end|>\n<|im_start|>assistant\n"
+        cls.prompt_video = "user\n<video>\nWhat do you see in this video?<|im_end|>\n<|im_start|>assistant\n"
```

Comment:
This is an Integration tests, not decorated by @slow, so the `setupClass` would be called on CircleCI.

Again, my suggestion is to focus on what we want to do with this PR: reduce calls to Hub on **CircleCI**

Reply:
Reverted this too!